common/cache/lru.go

```diff
@@ -152,7 +152,7 @@ func New(maxSize int, opts *Options) Cache {
 
 	return &lru{
 		byAccess:   list.New(),
-		byKey:      make(map[interface{}]*list.Element, opts.InitialCapacity),
+		byKey:      make(map[interface{}]*list.Element),
 		ttl:        opts.TTL,
 		maxSize:    maxSize,
```

Comment:
not for this PR, but if I were writing code using this, I'd want to control both max total size and max element size separately, so that I can prevent one element from using the entire cache space. just an idea for the future

Reply:
agree, I think is future improvement for the cache.